### PR TITLE
Added acceptance criteria from the spec-development-document

### LINKED ISSUE #905: Update 0062-SPAM-spam_protection - post Oregon Trail
```
0062-SPAM-spam_protection.md
  Criteria:      0
  Referenced by: 0
```

### PR DESCRIPTION
**Pull request by Vegaklaus**
closes #905 .

### CODE REVIEW

File: protocol/0062-SPAM-spam_protection.md [hunk 1]
```diff
@@ -88,3 +88,31 @@ is then not increased for another 10 blocks. At the beginning of every epoch, th
   banning policy this is not doable from one account, but with a sybil attack it can be done. If this ends up being a
   problem, we can address it by increasing the ban-time.
   
+### Acceptance Criteria
```
Comment (gordsport) [inline protocol/0062-SPAM-spam_protection.md:93]: @edd Please can you advise @Vegaklaus  in the addition of the AC codes in here so we can ensure this is picked up when covered in tests.

File: protocol/0062-SPAM-spam_protection.md [hunk 2]
```diff
@@ -88,3 +88,31 @@ is then not increased for another 10 blocks. At the beginning of every epoch, th
...
+
+ - A spam attack using votes/governance proposals is detected and the votes transactions are rejected, i.e.,
+This conversation was marked as resolved by Vegaklaus
+ Show conversation
```
Comment (gordsport) [inline protocol/0062-SPAM-spam_protection.md]: Can the following be removed:
"This conversation was marked as resolved by Vegaklaus
 Show conversation"

**Review by gordsport — CHANGES_REQUESTED** [review]

File: protocol/0062-SPAM-spam_protection.md (hunk 1 above)
Comment (edd) [reply protocol/0062-SPAM-spam_protection.md:93]: I’ll go ahead and do it, then explain the logic! It’s outline here: https://github.com/vegaprotocol/approbation. One day I’d like to make it automatic. Not today though.

Comment (edd) [reply protocol/0062-SPAM-spam_protection.md:93]: So! Every acceptances criteria that could be an individual, testable thing is assigned a code. The codes are generated in the fashion described above, and put in as self-linking-links so they’re easy to copy and paste. Job done!

There were one or two that made me think QA might say ‘This isn’t testable’. We’ll remove them when someone says that!

**Review by edd — APPROVED** [review]
I assigned AC codes, ands removed the bit @gordsport wanted to be removed

**Review by gordsport — APPROVED** [review]

**Pull request merged**